<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>900</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1600</width>
    <height>900</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>御坂下载器</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/icon/bilibili_logo.svg</normaloff>:/icon/bilibili_logo.svg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="mainLayout" columnstretch="2,10">
      <item row="0" column="1">
       <layout class="QVBoxLayout" name="mainMenuLayout">
        <item>
         <widget class="QStackedWidget" name="stackedWidget"/>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="mainContentLayout">
        <item>
         <widget class="QToolButton" name="loginBtn">
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">QToolButton{border:none;}

</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="res.qrc">
            <normaloff>:/icon/user.svg</normaloff>:/icon/user.svg</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>80</width>
            <height>80</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="autoExclusive">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="homeBtn">
          <property name="styleSheet">
           <string notr="true">QToolButton{border:none;}
/* 鼠标悬停 */
QToolButton:hover{
	background-color: rgb(246, 246, 246);
}
/* 点击和按下 */
QToolButton:pressed,QToolButton:checked{
	background-color: rgb(205, 205, 205);	
}
</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="res.qrc">
            <normaloff>:/icon/home.svg</normaloff>:/icon/home.svg</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>80</width>
            <height>80</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="autoExclusive">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="downloadBtn">
          <property name="styleSheet">
           <string notr="true">QToolButton{border:none;}
/* 鼠标悬停 */
QToolButton:hover{
	background-color: rgb(246, 246, 246);
}
/* 点击和按下 */
QToolButton:pressed,QToolButton:checked{
	background-color: rgb(205, 205, 205);	
}
</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="res.qrc">
            <normaloff>:/icon/download.svg</normaloff>:/icon/download.svg</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>80</width>
            <height>80</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="autoExclusive">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolsBtn">
          <property name="styleSheet">
           <string notr="true">QToolButton{border:none;}
/* 鼠标悬停 */
QToolButton:hover{
	background-color: rgb(246, 246, 246);
}
/* 点击和按下 */
QToolButton:pressed,QToolButton:checked{
	background-color: rgb(205, 205, 205);	
}
</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="res.qrc">
            <normaloff>:/icon/CodeSandbox.svg</normaloff>:/icon/CodeSandbox.svg</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>80</width>
            <height>80</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="autoExclusive">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="settingBtn">
          <property name="styleSheet">
           <string notr="true">QToolButton{border:none;}
/* 鼠标悬停 */
QToolButton:hover{
	background-color: rgb(246, 246, 246);
}
/* 点击和按下 */
QToolButton:pressed,QToolButton:checked{
	background-color: rgb(205, 205, 205);	
}
</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="res.qrc">
            <normaloff>:/icon/setting.svg</normaloff>:/icon/setting.svg</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>80</width>
            <height>80</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="autoExclusive">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="aboutBtn">
          <property name="styleSheet">
           <string notr="true">QToolButton{border:none;}
/* 鼠标悬停 */
QToolButton:hover{
	background-color: rgb(246, 246, 246);
}
/* 点击和按下 */
QToolButton:pressed,QToolButton:checked{
	background-color: rgb(205, 205, 205);	
}
</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="res.qrc">
            <normaloff>:/icon/info-circle.svg</normaloff>:/icon/info-circle.svg</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>80</width>
            <height>80</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="autoExclusive">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1600</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
